internal/backend/gdrive/gdrive.go

```diff
@@ -0,0 +1,566 @@
+package gdrive
+
+// TODO logging
```

Comment:
What are these comments? Are you not done yet? IMHO inline TODO comments are a code smell (yes, we have them in the code base, but I'd very much try to get rid of the comments ASAP).